desktop/cmp/contextmenu/StoreContextMenu.js

```diff
@@ -45,6 +48,26 @@ export class StoreContextMenu {
                         colChooserModel.open();
                     }
                 });
+            case 'exportExcel':
+                return new StoreContextMenuItem({
+                    text: 'Export to Excel',
+                    icon: Icon.download(),
+                    hidden: !gridModel || !gridModel.enableExport,
+                    disabled: !gridModel.store.count,
```

Comment:
If gridModel can truly be null here (and the doc says that it can) line 56 and 66 are going to throw DNP